src/commands/wasm2obj.rs

```diff
@@ -47,16 +42,14 @@ pub struct WasmToObjCommand {
 
 impl WasmToObjCommand {
     /// Executes the command.
-    pub fn execute(&self) -> Result<()> {
-        self.handle_module()
-    }
-
-    fn handle_module(&self) -> Result<()> {
-        if self.common.log_to_files {
-            let prefix = "wasm2obj.dbg.";
-            init_file_per_thread_logger(prefix);
-        } else {
-            pretty_env_logger::init();
+    pub fn execute(self) -> Result<()> {
+        if !self.common.disable_logging {
+            if self.common.log_to_files {
+                let prefix = "wasm2obj.dbg.";
+                init_file_per_thread_logger(prefix);
+            } else {
+                pretty_env_logger::init();
+            }
```

Comment:
I've seen this in a few places, could this perhaps be refactored to a common function in `src/lib.rs`?

Reply:
I'll extract this out.